kolibri/content/utils/import_export_content.py

```diff
@@ -61,3 +77,40 @@ def get_num_coach_contents(contentnode, filter_available=True):
             .count()
     else:
         return 1 if contentnode.coach_content else 0
+
+
+def retry_import(command, e, name, f=None, overall_progress_update=None):
+    logging.error("An error occured during channel import: {}".format(e))
+
+    # When there is an Internet connection error or timeout error,
+    # or HTTPError where the error code is one of the RETRY_STATUS_CODE,
+    # return False, 0 to retry the file transfer, or return True, 0 to
+    # indicate the cancellation
+    if (
+            isinstance(e, ConnectionError) or
+            isinstance(e, Timeout) or
+            (isinstance(e, HTTPError) and e.response.status_code in RETRY_STATUS_CODE) or
+            (isinstance(e, SSLERROR) and 'decryption failed or bad record mac' in str(e))):
+        return sleep_before_retry(command), 0
+
+    # Skip the file if it does not exist on the server or disk.
+    # This only applies to content import not channel import.
+    elif (
+            name == 'content' and
```

Comment:
Can we remove the whole `name` parameter and instead use deliberate `skip_404=True` kwarg?

Reply:
yes I can do that!